Automattic/gridicons
# Build: build PDF using `svg-to-pdfkit`

![screen shot 2017-07-14 at 15 31 13](https://user-images.githubusercontent.com/4389/28216402-838d48ce-68a9-11e7-9a8b-876a5cfb3997.png)

This PR adds the `svg2pdf` task using [svg-to-pdfkit](https://www.npmjs.com/package/svg-to-pdfkit) (which uses [PDFKit](http://pdfkit.org/)).

The scope of the PDF is to be used in iOS, so we ran a couple of tests with @frosty.

PDF in iOS 10:

![pasted image at 2017_07_14 03_14 pm](https://user-images.githubusercontent.com/4389/28216548-fe0e88ec-68a9-11e7-8265-1db27ec0215a.png)

PDF converted with Paintcode:

![pasted image at 2017_07_14 03_26 pm](https://user-images.githubusercontent.com/4389/28216559-0921288e-68aa-11e7-9a7d-216cbb5eee16.png)



### To test

* Run `grunt` and make sure everything gets generated as usual
* Try emptying the `pdf` folder and run `grunt` again, all the PDFs should appear
* Check that the PDFs shapes are correct

/cc @mattmiklic 

Props: Massimo Menichinelli @openp2pdesign (tipped me [here](https://www.facebook.com/folletto/posts/10158960582480623?comment_id=10158960761175623&comment_tracking=%7B%22tn%22%3A%22R0%22%7D))
